content/widgets/reverseHighlight.md

```diff
@@ -0,0 +1,17 @@
+---
+title: ReverseHighlight
+type: widget
+html: |
+  <span class="ais-ReverseHighlight><span class"ais-ReverseHighlight-nonHighlighted">This is the</span> <mark class="ais-ReverseHighlight-highlighted">reversed highlighted text</mark></span>
```

Comment:
```suggestion
  <span class="ais-ReverseHighlight"><span class"ais-ReverseHighlight-nonHighlighted">This is the</span> <mark class="ais-ReverseHighlight-highlighted">reversed highlighted text</mark></span>
```

Reply:
Looks like my vim extension messed up again  on the replace 🤔